### PR TITLE
Launch V1 of top-level navigation

### PR DESCRIPTION
**Pull request by ghost**
Closes department-of-veterans-affairs/vets.gov-team#593

- Updated Veterans Crisis line styles (closes department-of-veterans-affairs/vets.gov-team#991)
- Rewrote crisis line media queries to use USWDS media mix-in
- Updated mobile search button design (closes department-of-veterans-affairs/vets.gov-team#994)

### CODE REVIEW

**Comment by emdashed** [comment]
This looks awesome I am so so so so excited!

I think this should be the same height across:
screen shot 2017-02-13 at 4 13 34 pm


And wanting to make sure we won't lose the menu as it is now on this view (same is true for FL)?
screen shot 2017-02-13 at 4 13 49 pm

**Comment by melwoodard** [comment]
I see some smaller visual bugs that I can log separately for a follow-up. I think Emily's screenshot above of the nav not being on every page is definitely a blocker, as is how the menu appears on mobile:

(iphone 5 / iOS10 / Safari)

**Review by melwoodard — DISMISSED** [review]
The desktop menu height and mobile menu layout are looking much better - however scrolling within the menu on mobile is locked for me (same device/software specs as before).

**Comment by emdashed** [comment]
Adding to @melwoodard comment - menu works fine on my android. I can't scroll but I don't see a need to because everything fits for me. horizontally it loads the desktop version (which is best case for now but we may need to revisit later)

**Comment by melwoodard** [comment]
Yep, the initial view is fine, but digging into submenus is where it breaks down for us small-screeners.

File: src/js/legacy/mega-menu.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+      this.closeMenu(event);
+    } else {
+      this.closeAll();
+      this.openMenu(event);
+
+      /*
+      Open the first sub-menu and expand first trigger 
+      when the breakpoint > 768
+      */
+      if(matchMedia('(min-width: 768px)').matches && whichMenu.querySelector('.vetnav-panel--submenu')){
```
Comment (jbalboni) [inline src/js/legacy/mega-menu.js]: Can you use `isWideScreen` here?

File: src/js/legacy/mega-menu.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  }
+
+  // Make the document body unscrollable when menu is open
+  document.body.classList.toggle('va-pos-fixed');
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  const mm = new MegaMenu(document.querySelector('#vetnav-menu'));
+  
+  if(mm.isWideScreen()) {
```
Comment (jbalboni) [inline src/js/legacy/mega-menu.js]: I think you need to do these calls in a resize handler (or some other way). If I resize my screen across the breakpoint, the desktop menu doesn't display (going from mobile -> desktop) and the menu button click does nothing (desktop -> mobile).

File: src/js/legacy/mega-menu.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+    target.setAttribute('aria-expanded', true);
+    this.getMenu(menu).removeAttribute('hidden','hidden');
+  }
+
+  toggleMenu(event) {
+    const eTarget = event.target;
+    const whichMenu = this.getMenu(eTarget.getAttribute('aria-controls'));
+
+    if(eTarget.getAttribute('aria-expanded') == 'true') {
```
Comment (jbalboni) [inline src/js/legacy/mega-menu.js]: Is this file getting linted? I figured eslint would flag the `==`.

**Review by jbalboni — COMMENTED** [review]
Looks nice!

**Review by crwallace — APPROVED** [review]
This looks great! From an a11y standpoint this is easy to navigate from the keyboard and all of the button text is clear. I will leave the visual comments to @emdashed and @melwoodard

**Comment by bethpotts** [comment]
@annekainicUSDS Are you able to help with any of this or could you direct me to the right person? The first three below have to do with links not going anywhere and the last one is a content change--capitalization issues. Thanks!:

1)    Explore benefits > Health care > Apply now (green button)
a.    The green button takes me to “Sorry—we can’t find that page”

2)    Explore benefits > Education & training > Compare GI Bill benefits
a.    Compare GI Bill benefits takes me to “Sorry—we can’t find that page”

3)    Explore benefits > Education & training > Apply now (green button)
a.    The green button takes me to “Sorry—we can’t find that page”

4)    Explore benefits > Careers & employment
a.    Find a job / Hire a veteran (please capitalize the V in “veteran”)
b.    Resources for military and veteran family members (please capitalize the V in “veteran”)

**Comment by tiffanybbrown-va** [comment]
@bethpotts the first three items will fail locally unless you have the `vets-api` repo installed and running. The paths, however, are correct. `http://localhost/education/apply-for-education-benefits/application/1990/introduction` will break, but when it's on vets.gov, it will point to `https://www.vets.gov/education/apply-for-education-benefits/application/1990/introduction` which is correct.

**Comment by emdashed** [comment]
@robbiethegeek-usds  or @crwallace - who is up for testing this in IE? I want to explicitly verify that this is being looked at in IE by someone before ship it.

**Comment by jcmeloni-usds** [comment]
I can test it in IE11 if it's on dev or staging, if that's helpful!

**Comment by emdashed** [comment]
@jcmeloni and everyone! Please don't ship anything that hasn't been tested thoroughly in IE. 😨

**Comment by tiffanybbrown-va** [comment]
Just took a look at it in IE11. There was a small background color issue that should be fixed in this latest round of changes.

**Comment by mnorthuis** [comment]
The hover state on the buttons in each drop menu should not have an underline.  This should be corrected on the following buttons:
Health care > Apply now
Disability > Go to eBenefits to apply
Education > Apply now

**Comment by bethpotts** [comment]
@emdashed Who can fix the capping? Just don't want that to get lost... thanks!
Explore benefits > Careers & employment
a. Find a job / Hire a veteran (please capitalize the V in “veteran”)
b. Resources for military and veteran family members (please capitalize the V in “veteran”)

**Comment by jcmeloni-usds** [comment]
@melwoodard @emdashed Which of you will be giving final signoff on pushing this to production?  It seems there are text changes that conflict with the original guidance (which is fine) but we need to know if that's all of them and who is giving the final ok to merge this in and release today (or hold for tomorrow, etc).  Thanks!

**Comment by emdashed** [comment]
@jcmeloni @tiffanybbrown-va we always capitalize Veteran. I don't know if that warrants a "text change", but please let's keep it consistent! I'm happy to give the final sign off, and I assume @tiffanybbrown-va will let me know when she's ready? We chatted yesterday, so stayin' on it. Thanks @jcmeloni and @tiffanybbrown-va and good catch @bethpotts !

**Comment by jcmeloni-usds** [comment]
@emdashed Cool, I'm glad someone is going to give a final 👍 when it's time.  

I just meant that if Tiffany took the directions in the original ticket literally, then she would literally have used a small v because that's what it says, so she'll have to push another commit to capitalize it....which is not a big deal, obvs, just wanted to make sure that's the directive that needs to be addressed.

**Comment by jcmeloni-usds** [comment]
Hooray capitalization!

**Comment by emdashed** [comment]
Just looking now, the biggest issue I'm seeing is that the navigation completely disappears for me on middle-ish sizes (tablet, horizontal phone) I think this is not blocking, but nonetheless should be fixed soon. will add it as an issue elsewhere, and giving a 👍 for now. 

Note that I am not checking across browsers, since I have been told this has happened.

screenshot_20170215-080617
screenshot_20170215-080529

@tiffanybbrown-va and co THIS LOOKS SO AWESOME I AM SO EXCITED!!!

**Review by emdashed — APPROVED** [review]

**Comment by jcmeloni-usds** [comment]
@emdashed awesome, thanks

**Pull request merged**